server/graphql/v2/query/collection/ActivitiesCollectionQuery.ts

```diff
@@ -67,15 +67,18 @@ const ActivitiesCollectionQuery = {
     const include = [];
     for (const account of accounts) {
       if (isRootUser || req.remoteUser.isAdminOfCollective(account)) {
+        // Include all activities related to the account itself
         accountOrConditions.push({ CollectiveId: account.id }, { FromCollectiveId: account.id });
+
+        // Include all activities related to the account's children
         if (args.includeChildrenAccounts) {
           accountOrConditions.push({ '$Collective.ParentCollectiveId$': account.id });
-          include.push({
-            model: models.Collective,
-            attributes: [],
-            required: true,
-          });
+          if (include.length === 0) {
```

Comment:
@Betree : Don't quite understand why we need this condition? I think `include` is always empty.

Reply:
@SudharakaP No, because we're in a loop. The previous version of the code pushed a new `include` for each account, to 10 accounts resulted in 10 includes

Reply:
Ops, I didn't see the loop. 🤯